CHANGES.rst

```diff
@@ -40,7 +40,9 @@
 - HIRES wavelength solution improvements galor
 - Added `redo_slits` option
 - Refactored ``load_line_lists()`` yet again!
+- Introduced PCA method for telluric corrections
 
+>>>>>>> develop
```

Comment:
Looks like you generated conflicts when you merged. Please remove the >>>>>> develop.

Reply:
Done. For some reason merging in CHANGES always gives me strange conflicts...